### PR TITLE
Gen3 Monthly Release 2023.06 qa-heal.planx-pla.net 1687484045

### PR DESCRIPTION
**Pull request by PlanXCyborg**
Applying version 2023.06 to qa-heal.planx-pla.net

### CODE REVIEW

File: qa-heal.planx-pla.net/manifest.json [hunk 1]
```diff
@@ -8,28 +8,28 @@
...
+    "arborist": "707767160287.dkr.ecr.us-east-1.amazonaws.com/gen3/arborist:2023.06",
+    "audit-service": "707767160287.dkr.ecr.us-east-1.amazonaws.com/gen3/audit-service:2023.06",
     "aws-es-proxy": "707767160287.dkr.ecr.us-east-1.amazonaws.com/gen3/aws-es-proxy:v1.3.1",
     "cedar-wrapper": "quay.io/cdis/cedar-wrapper-service:0.2.0",
-    "dashboard": "707767160287.dkr.ecr.us-east-1.amazonaws.com/gen3/gen3-statics:2023.04",
-    "fence": "707767160287.dkr.ecr.us-east-1.amazonaws.com/gen3/fence:2023.04",
+    "dashboard": "707767160287.dkr.ecr.us-east-1.amazonaws.com/gen3/gen3-statics:2023.06",
+    "fence": "707767160287.dkr.ecr.us-east-1.amazonaws.com/gen3/fence:2023.06",
     "fluentd": "fluent/fluentd-kubernetes-daemonset:v1.15.3-debian-cloudwatch-1.0",
     "frontend-framework": "quay.io/cdis/frontend-framework:0.4.5",
```
Comment (mfshao) [inline qa-heal.planx-pla.net/manifest.json:18]: can we deploy 0.4.6 for this

File: qa-heal.planx-pla.net/manifest.json [hunk 2]
```diff
@@ -8,28 +8,28 @@
...
-    "revproxy": "707767160287.dkr.ecr.us-east-1.amazonaws.com/gen3/nginx:2023.04",
-    "sheepdog": "707767160287.dkr.ecr.us-east-1.amazonaws.com/gen3/sheepdog:2023.04",
-    "sower": "707767160287.dkr.ecr.us-east-1.amazonaws.com/gen3/sower:2023.04",
-    "ssjdispatcher": "707767160287.dkr.ecr.us-east-1.amazonaws.com/gen3/ssjdispatcher:2023.04",
-    "wts": "707767160287.dkr.ecr.us-east-1.amazonaws.com/gen3/workspace-token-service:2023.04"
+    "manifestservice": "707767160287.dkr.ecr.us-east-1.amazonaws.com/gen3/manifestservice:2023.06",
+    "metadata": "707767160287.dkr.ecr.us-east-1.amazonaws.com/gen3/metadata-service:2023.06",
+    "peregrine": "707767160287.dkr.ecr.us-east-1.amazonaws.com/gen3/peregrine:2023.06",
+    "pidgin": "707767160287.dkr.ecr.us-east-1.amazonaws.com/gen3/pidgin:2023.06",
+    "portal": "707767160287.dkr.ecr.us-east-1.amazonaws.com/gen3/data-portal:2023.06",
```
Comment (mfshao) [inline qa-heal.planx-pla.net/manifest.json:26]: and don't change this

**Review by hughestr — DISMISSED** [review]

File: qa-heal.planx-pla.net/manifest.json (hunk 1 above)
Comment (haraprasadj) [reply qa-heal.planx-pla.net/manifest.json]: done

File: qa-heal.planx-pla.net/manifest.json (hunk 2 above)
Comment (haraprasadj) [reply qa-heal.planx-pla.net/manifest.json]: done

**Pull request merged**